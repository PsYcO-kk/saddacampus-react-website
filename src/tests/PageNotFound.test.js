import {shallow} from 'enzyme';
import React from 'react';
import PageNotFound from '../components/NotFoundPage.js'

test("testing for page not found component" , ()=>{
    const wrapper = shallow(<PageNotFound />)
    expect(wrapper).toMatchSnapshot();
})
