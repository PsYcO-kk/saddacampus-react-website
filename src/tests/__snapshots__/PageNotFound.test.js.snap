// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for page not found component 1`] = `<div />`;
